Quarterly Checkins/QC-2019-Q1.md

```diff
@@ -0,0 +1,40 @@
+# Quarterly Check-in 2019 Q1
+
+## Summary
+
+|Category| Incoming | Outgoing |
+| --- | :---: |  :---: | 
+| Total issues | 36 | 30 |
+| Enhancements | 8 | 5 |
+| Bugs | 6 | 6 |
+
+## Work accomplished
+
+The following table highlights important enhancements and bug fixes done in MSTestV2 in the given period.
+
+| Issue | Remarks  |
+| --- | -- |
+| [#563](https://github.com/Microsoft/testfx/pull/563) | Made IDictionary properties in TestContext to be consistent between netframework and netcore. |
+| [#585](https://github.com/Microsoft/testfx/pull/585) | Added support for CancellationTokenSource in TestContext to help in timeout scenario. |
```

Comment:
nit can we sort this based on when the change was done

Reply:
This is sorted based on importance of issues. We follow similar pattern for our release notes and people are accustomed to this.